addons/gamification/tests/test_challenge.py

```diff
@@ -137,6 +139,57 @@ def test_20_update_all_goals_filter(self):
             "Only portal user last logged in before last challenge update should not have been updated.",
         )
 
+    def test_30_close_goals(self):
+        Goals = self.env['gamification.goal']
+
+        admin_user = self.env.ref('base.user_admin')
+        demo_user = self.robot
```

Comment:
Why two references to the same user?
```suggestion
```

Reply:
IIRC there was a CI error or warning about depending on demo data, which may not be available. I must've "fixed" that issue with this brain fart. I'll fix this.